### DISCUSSION TITLE
Enabling sshd?

### DISCUSSION THREAD

**Discussion by nrvale0** [Vanilla-OS/live-iso#381] [Q&A] [answered]
I've been trying to enable sshd post-install. It looks like the preferred method of installing sshd is:

```sudo apt install task-ssh-server```

however I don't get a started sshd and if I try to do that myself with:

```sudo systemctl enable ssh``` 

I also don't get a running daemon ever after reboot.

```sudo systemctl enable --now ssh```  fails and 'journalctl' reports:

```
Nov 04 01:50:24 vanilla-test sshd[618]: error: Bind to port 22 on 0.0>
Nov 04 01:50:24 vanilla-test sshd[618]: error: Bind to port 22 on :: >
Nov 04 01:50:24 vanilla-test sshd[618]: fatal: Cannot bind any addres>
```

There is nothing listening on tcp/22 according to 'ss'.

Advice?

**Comment by taukakao** [comment, marked as answer]
sshd is already installed on the host system.

Either:
1. Go to Settings -> System -> Secure Shell -> Enable
or
2. `host-shell systemctl enable --now sshd`

**Reply by nrvale0** [reply]
Ty. That did it.
